storage/bootstrap.go

```diff
@@ -110,7 +110,8 @@ func (m *bootstrapManager) targetRanges(at time.Time) xtime.Ranges {
 	midPoint := at.
 		Add(-ropts.BlockSize()).
 		Add(-ropts.BufferPast()).
-		Truncate(ropts.BlockSize())
+		Truncate(ropts.BlockSize()).
+		Add(ropts.BlockSize())
```

Comment:
So after looking at logs, because we bootstrap using the "from" inclusive and "to" exclusive - it meant that we weren't properly bootstrapping everything up to the latest block for the FS bootstrapper.  We were bootstrapping up to just before the latest block in the first pass with the FS bootstrapper, then would do the latest flushed block from FS bootstrapper and the current block from Peers bootstrapper.  

This is not what we want, this change corrects this behavior.